composer/callbacks/export_for_inference.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Callback to export model for inference."""
+
+from __future__ import annotations
+
+import logging
+from typing import Callable, Optional, Sequence, Union
+
+import torch.nn as nn
+
+from composer.core import State
+from composer.core.callback import Callback
+from composer.loggers import Logger
+from composer.utils.inference import ExportFormat, export_for_inference
+from composer.utils.object_store import ObjectStore
+
+log = logging.getLogger(__name__)
+
+__all__ = ['ExportForInference']
+
+
+class ExportForInference(Callback):  # noqa: D101
```

Comment:
Since we also have the `export_for_inference` method, this might be worth being a bit more verbose, e.g. `ExportForInferenceCallback`

Reply:
Changed!